common/src/main/java/com/wynntils/models/skillpoint/SkillPointModel.java

```diff
@@ -0,0 +1,388 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.skillpoint;
```

Comment:
I'm thinking this belongs in `models.character`...

Reply:
Honestly I think we already have too much irrelevant stuff in there... At first I was under the impression that CharacterModel meant what we would previously call ClassModel, just the data about the player's currently selected class. But then slowly we added death location, Silverbull, veteran stuff, etc.

Reply:
You keep the model separate, just move the package, as it's still character related. That way the code is not getting messier. At least, I think this is what magicus meant.

Reply:
Yes. While we often has the format "models.foo" contains "FooModel", I don't think this is really necessary. The package grouping is on a somewhat higher level, and can contain many independent, but related, models.

I thought the CharacterModel contained info about our current skill points, but maybe I remembered wrong? The skill point loadout should be located next to the model containing info on our current skill points anyway. If it is not the CharacterModel, then whatever model it is.

Reply:
> I thought the CharacterModel contained info about our current skill points, but maybe I remembered wrong?

There is only a FIXME about adding support for it :)

Reply:
*lol* 

Then maybe that fixme can be removed now? Or, at least if the skillpoint model is flexible enough to allow us to read the current values of our skill points. (I'm thinking they should basically be exported by the model so they can e.g. be available as functions).